packaging/build/appveyor.ps1

```diff
@@ -87,10 +89,10 @@ Function Build-Wheel($python_path) {
     }
 
     # ensure pip is current (some appveyor pips are not)
-    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --upgrade pip }
+    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --no-warn-script-location --upgrade pip }
 
     # ensure required-for-build packages are present and up-to-date
-    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --upgrade cython wheel setuptools --no-warn-script-location }
+    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --no-warn-script-location --upgrade cython wheel setuptools --no-warn-script-location }
```

Comment:
This is doing `--no-warn-script-location` twice.

Should we really be suppressing deprecation warnings?